blocks/cards/cards.css

```diff
@@ -174,19 +215,19 @@ main .cards.overlay > ul > li p.button-container a {
 }
 
 /* champions */
-main .cards.cards-champions h3 {
+main .cards.champions h3 {
   margin: 0;
 }
 
 @media (min-width: 900px) {
-  main .cards.cards-champions > ul {
+  main .cards.champions > ul {
     grid-template-columns: repeat(auto-fill, minmax(calc((100% / 2) - 30px), 1fr));
     grid-gap: 30px;
   }
 }
 
 @media (min-width: 700px) {
-  main .cards.cards-champions > ul > li img {
```

Comment:
simplifying variant name for authors ("champion" vs. "cards-champion"), saving a whole six characters